Sources/SectionScrubber.swift

```diff
@@ -152,66 +156,62 @@ public class SectionScrubber: UIView {
         }
     }
 
-    func calculateYPosInView(forYPosInContentView yPosInContentView: CGFloat) -> CGFloat{
-        let percentageInContentView = yPosInContentView / containingViewContentSize.height
-        return (containingViewFrame.height * percentageInContentView ) + containingViewFrame.minY
+    func calculateYPosInView(forYPosInContentView yPosInContentView: CGFloat) -> CGFloat {
+        let percentageInContentView = yPosInContentView / self.containingViewContentSize.height
+        return (containingViewFrame.height * percentageInContentView) + self.containingViewFrame.minY
     }
 
     func calculateYPosInContentView(forYPosInView yPosInView: CGFloat) -> CGFloat {
         let percentageInView = (yPosInView - containingViewFrame.minY) / containingViewFrame.height
         return (containingViewContentSize.height * percentageInView) - containingViewFrame.minY
     }
 
-    func handleScrub(gestureRecognizer: UIGestureRecognizer){
-            self.sectionLabelState = gestureRecognizer.state == .Ended ? .Hidden : .Visible
-            self.userInteractionOnScrollViewDetected()
-
-
-        if let panGestureRecognizer = gestureRecognizer as? UIPanGestureRecognizer {
+    func handleScrub(gestureRecognizer: UIGestureRecognizer) {
+        self.sectionLabelState = gestureRecognizer.state == .Ended ? .Hidden : .Visible
+        self.userInteractionOnScrollViewDetected()
 
-            if panGestureRecognizer.state == .Began || panGestureRecognizer.state == .Changed {
-                let translation = panGestureRecognizer.translationInView(self)
-                var newYPosForSectionScrubber =  self.frame.origin.y + translation.y
+        if let panGestureRecognizer = gestureRecognizer as? UIPanGestureRecognizer where panGestureRecognizer.state == .Began || panGestureRecognizer.state == .Changed {
 
+            let translation = panGestureRecognizer.translationInView(self)
+            var newYPosForSectionScrubber = self.frame.origin.y + translation.y
 
-                if newYPosForSectionScrubber < containingViewFrame.minY {
-                    newYPosForSectionScrubber = containingViewFrame.minY
-                }
+            if newYPosForSectionScrubber < containingViewFrame.minY {
+                newYPosForSectionScrubber = containingViewFrame.minY
+            }
 
-                if newYPosForSectionScrubber > containingViewFrame.height + 60.8{
-                    newYPosForSectionScrubber = containingViewFrame.height + 60.8
-                }
+            if newYPosForSectionScrubber > containingViewFrame.height + 60.8 {
+                newYPosForSectionScrubber = containingViewFrame.height + 60.8
```

Comment:
Oh shit... There's my bug! I'll have to fix this in a more solid way. Slipped through because of holidays.